spring-cloud-contract-wiremock/src/main/java/org/springframework/cloud/contract/wiremock/file/ResourcesFileSource.java

```diff
@@ -170,7 +183,7 @@ public boolean exists() {
 	}
 
 	@Override public void deleteFile(String name) {
-
+		throw new UnsupportedOperationException("Resource file sources are read-only");
```

Comment:
You want to merge this to `1.1.x` but it's a breaking change. Let's not throw this exception

Reply:
Ok, I will revert this change.